x-pack/docs/build.gradle

```diff
@@ -5,16 +5,16 @@ apply plugin: 'elasticsearch.rest-resources'
  * `// CONSOLE` and `// TESTRESPONSE` but have yet to be converted. Try and
  * only remove entries from this list. When it is empty we'll remove it
  * entirely and have a party! There will be cake and everything.... */
-tasks.named("buildRestTests").configure {
-  expectedUnconvertedCandidates = [
-          'en/rest-api/watcher/put-watch.asciidoc',
-          'en/security/authentication/user-cache.asciidoc',
-          'en/security/authorization/run-as-privilege.asciidoc',
-          'en/security/ccs-clients-integrations/http.asciidoc',
-          'en/rest-api/watcher/stats.asciidoc',
-          'en/watcher/example-watches/watching-time-series-data.asciidoc',
-  ]
-}
+// tasks.named("buildRestTests").configure {
```

Comment:
Should we remove this? in fact, can we remove the entire `x-pack/docs` project?

Reply:
Sorry — I left a sentence about this in the top-level comment, but I should have left one here as well.

The `x-pack/docs` directory is currently required for the docs build. I left these files here so that we don't break the build. I'll remove the `x-pack/docs` directory and these files after I merge this PR and update the docs build config.

If wanted, I can replace this and the other files in the `x-pack/docs` directory with a placeholder txt file. The docs build only need the directory to exist — even if it doesn't pull anything from it.

Reply:
👍